### DISCUSSION TITLE
Is there a way to bind a comment to a block?

### DISCUSSION THREAD

**Discussion by OriLiMu** [ScratchAddons/ScratchAddons#7599] [Help] [answered]
I am making a find-reference addon.  for example in this case, I want to find the usage of variable x. I think it will be much more useful if I can bind a comment to the "when_flag_clicked" big block. So I can know the purpose of the big block
![image](https://github.com/ScratchAddons/ScratchAddons/assets/11295355/18c93d55-6097-4134-a85c-881b7c4b20ee)

But seems the comment cant bind to block in the editor. So any way to make this?

**Comment by ghost** [comment, marked as answer]
Scratch already has a feature for this - Rightclick a block and click add comment.
